.buildkite/scripts/generate_xpack_packetbeat_pipeline.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+source .buildkite/scripts/common.sh
+
+set -euo pipefail
+
+pipelineName="pipeline.xpack-metricbeat-dynamic.yml"
```

Comment:
name is not packetbeat
```suggestion
pipelineName="pipeline.xpack-packetbeat-dynamic.yml"
```